doc/source/WorkingWithScenes/AnatomyOfACamera/index.md

```diff
@@ -0,0 +1,136 @@
+
+```eval_rst
+.. figure:: images/illustrationDepthOfField.png
+    :scale: 100%
+    :alt: Depth of field blur in Gaffer
+
+    **Figure 3.** An approximation of depth of field blur in Gaffer.
+```
+
+> Important :
+> The term _lens aperture_ has a meaning near to its real-world counterpart – it really can be thought of as a round opening in space.
+
+The fStop and focalDistance parameters are identical to f-number and focus distance in real lenses. They allow easy replication of real lens settings.
+
+The third parameter, focalLengthWorldScale, sets the scale between the camera's unit of measure and the world space's unit of measure. Because the circle occupies 3D space, the circle must be measured in world space units. Complicating things is the fact that the world space units might not be measured in millimeters. The scene could use any scale – centimeter, meter, foot, kilometer, etc. Therefore, the scale factor between them must be defined.
+
+For depth of field, lens aperture is calculated with the formula: 
+
+```
+lens aperture = (focalLength × focalLengthWorldScale) / fStop
+```
+
+For example, assume a lens with a focalLength of 50mm, an fStop of 4, and a world space measured in centimeters (default for Alembic/USD). First, we need to scale the focal length from mm to cm. Using a focalLengthWorldScale of 0.1 yields `50mm × 0.1 = 0.5cm`. This is the lens aperture at its widest. Dividing by the fStop of 4 results in the stopped-down lens aperture of `0.5cm / 4 = 0.125cm`.
+
+
+## Cameras in the Scene ##
+
+
+### Camera data ###
+
+Within the <a href="../AnatomyOfAScene/index.html#scene-hierarchy">scene paradigm</a>, a camera, just like any other scene component, starts with a location in the scene hierarchy. To define a camera, the location must have the following data:
+
+- **Transform:** The vectors that define the position and orientation of the camera.
+- **Object:** A special camera object at the location. Instead of geometry, the object stores camera data, called parameters.
+    - **Parameters:** The crucial values that define a camera, such as the perspective type, field of view/aperture, and depth of field settings. If defined, a special kind of optional parameters, called **render overrides**, will supercede the scene's <a href="../AnatomyOfAScene/index.html#options">render options</a> during computation and rendering.<br>
+    ![Camera parameters in the Scene Inspector](images/interfaceCameraParameters.png)
+- **Sets:** A list of sets the location belongs to. By default, every camera is assigned to an automatic "Cameras" set, accessible in the API by the `__cameras` variable.<br>
+    ![Camera sets in the Scene Inspector](images/interfaceCameraSets.png)
+
+
+### Data flow ###
+
+For the most part, camera data follow the same rules as other scene data. The notable exception is in the handling of the scene's render settings.
```

Comment:
I don't like the implication that different rules are followed for the _data_, because that isn't the case. As an introduction, I think it makes more sense to say something like, "to perform a render, Gaffer combines information from the camera in the scene and the render settings in the globals.".

Reply:
Point taken, but see my comment above about the notion of "rendering" vs. using the OpenGL preview in the _Viewer_.